terra_sdk/core/wasm/data.py

```diff
@@ -0,0 +1,116 @@
+from terra_proto.cosmwasm.wasm.v1 import (
+    AccessType,
+    AccessConfigUpdate as AccessConfigUpdate_pb,
+    AccessTypeParam as AccessTypeParam_pb,
+    AccessConfig as AccessConfig_pb
+)
+import attr
+from terra_sdk.core.bech32 import AccAddress
+
+from terra_sdk.util.json import JSONSerializable
+from terra_sdk.core.msg import Msg
+
+__all__ = [
+    "AccessConfig",
+    "AccessConfigUpdate",
+    "AccessTypeParam"
+]
+
+
+
+@attr.s
+class AccessConfig(JSONSerializable):
+    
+    permission: AccessType = attr.ib()
+    """"""
+
+    address: AccAddress = attr.ib()
+    """"""
+
+    def to_amino(self) -> dict:
+        return {
+            "permission": self.permission,
+            "address": self.address
+        }
+    
+    def to_proto(self) -> AccessConfig_pb:
+        return AccessConfig_pb(
+            permission=self.permission,
+            address=self.address
+        )
+
+    @classmethod
+    def from_data(cls, data:dict) -> AccessConfig:
```

Comment:
@JSHan94 we need to convert from primitive types to AccessType and vice versa.

Reply:
yes. actually, core/wasm is mostly WIP